### ISSUE TITLE
Webview Process Terminated

### ISSUE THREAD

**Issue by anutting** [react-native-webview/react-native-webview#1767]
I am attempting to embed a Unity WebGL game with my react-native application and serve the game via this react-native-webview. The game runs pretty well at first, but them I get the Webview Process Terminated message and I need to restart the process using the onContentProcessDidTerminate prop. The event sent to that function doesn't have any information about what caused the crash. Is there a way to understand what's happening to cause this crash? Running on iOS. Thanks.

**Comment by jotatoledo** [comment]
Ill never understand why issues are simply closed with no resolution.
Having the same problem when embedding a SPA

**Comment by siniarski** [comment]
While this topic is closed, hopefully this helps anyone that stumbles in.

I was finding the same thing with an app I was working on however what I discovered was this only affected simulators and only certain ones. When using a physical device it would work as expected. I managed to find this by having a physical device of a simulator that was causing this issue and having the physical device work perfectly.
